compiler/rustc_feature/src/accepted.rs

```diff
@@ -361,6 +361,8 @@ declare_features! (
     (accepted, self_in_typedefs, "1.32.0", Some(49303)),
     /// Allows `Self` struct constructor (RFC 2302).
     (accepted, self_struct_ctor, "1.32.0", Some(51994)),
+    /// Shortern the tail expression lifetime
+    (accepted, shorter_tail_lifetimes, "1.79.0", Some(123739)),
```

Comment:
I think this should be `CURRENT_RUSTC_VERSION`, correct?

Reply:
Applied